src/Keboola/StorageApi/Client.php

```diff
@@ -2413,9 +2427,13 @@ public function apiPut($url, $data = null)
      */
     public function apiPutJson(string $url, array $data = [])
     {
-        return $this->request('put', $url, [
-            'json' => $data,
-        ]);
+        $options = [];
+        if (is_array($data) && empty($data)) {
+            $options['json'] = new stdClass();
```

Comment:
tomu nerozumim, proc tady vsude ma byt stdClass kdyz se to vsude pouziva jako nativni pole?

Reply:
Protože:

$data = [] ==> json:`[]`, ale mělo by to být `{}`
$data = ['x'=>1] => json:`{"x":1}`

Reply:
sorry, nastavil jsem blbe diff proti `master`, ale melo byt proti `KBC-2761-json-methods` - tyhle veci jsme resili uz tam...